static/src/javascripts/projects/common/views/experiments/quick-survey.html

```diff
@@ -0,0 +1,92 @@
+<style>
```

Comment:
This should be in a CSS file I think - unless someone has mentioned to do it this way? If it is in scss we can make use of the variables for colours, margins, fonts etc - I guess it is embedded in the html as a way of not delivering it to the user?